cdk/lib/elastic-search-monitor.ts

```diff
@@ -170,14 +170,14 @@ export class ElasticSearchMonitor extends GuStack {
     Double check the disk space used (curl -s 'localhost:9200/_cat/allocation?v') and access the logs to see if you can
     find any useful error messages ($ grep "ERROR" /var/log/syslog).`;
 
-    const fifteenPercentDiskSpaceInBytes = 743819616190;
+    const twentyPercentDiskSpaceInBytes = 991759488253;
```

Comment:
minor: could https://docs.aws.amazon.com/cdk/api/v2/docs/aws-cdk-lib.Size.html make this more accessible somehow? For example:

```ts
import { Size } from 'aws-cdk-lib';

const totalDiskSpace = Size.gibibytes(50);
const twentyPercentDiskSpaceInBytes = totalDiskSpace.toBytes() * 0.2;
```

(I've guessed at 50GB, that's most likely incorrect!).

Reply:
Nice idea - https://github.com/guardian/elastic-search-monitor/pull/107/commits/e4d124df05b6c0fe56a41d4324d82f9646c954af. 

It looks like our threshold was slightly out before too - I think it was based on the `total` value from Cerebro, which doesn't seem to line up with the info from AWS:

<img width="319" alt="image" src="https://github.com/guardian/elastic-search-monitor/assets/19384074/b8984806-dcaa-4757-bd0c-8783b0b619de">